crates/holochain_p2p/src/types/wire.rs

```diff
@@ -63,8 +63,8 @@ pub enum WireMessage {
     GetValidationPackage {
         action_hash: ActionHash,
     },
-    CountersigningAuthorityResponse {
-        signed_actions: Vec<SignedAction>,
+    CountersigningSessionNegotiation {
+        message: event::CountersigningSessionNegotiationMessage,
```

Comment:
since this constitutes a breaking protocol change, should the changelog reflect that? or are we only adding breaking change if someone has to update their wasm? was never quite clear on that

Reply:
sure